ui/src/app/users/table.tsx

```diff
@@ -0,0 +1,42 @@
+import { useContext, useState, Dispatch, SetStateAction } from "react"
+import { AsideContext } from "../aside"
+import { UserEntry } from "../types"
+import Row from "./row"
+
+type TableProps = {
+    users: UserEntry[]
+}
+
+export function UsersTable({ users: rows }: TableProps) {
+    const { isOpen: isAsideOpen, setIsOpen: setAsideIsOpen } = useContext(AsideContext)
+    return (
+        <div className="p-panel">
+            <div className="p-panel__header is-sticky">
+                <h4 className="p-panel__title">Users</h4>
+                <div className="p-panel__controls">
+                    {rows.length > 0 && <button className="u-no-margin--bottom p-button--positive" aria-label="add-csr-button" onClick={() => setAsideIsOpen(true)}>Create New User</button>}
+                </div>
+            </div>
+            <div className="p-panel__content">
+                <div className="u-fixed-width">
+                    <table id="csr-table" aria-label="Certificate Requests Table" className="p-table--expanding">
+                        <thead>
+                            <tr>
+                                <th>ID</th>
+                                <th>Username</th>
+                                <th>Delete</th>
```

Comment:
Can we take the same approach we have for CSR's and have a clickable menu icon which would lead to the Delete option. We'll likely want the Change Password option in there as well.

Reply:
Do we want the admin to be able to change the password for any user?

Reply:
made the actions menu anyway since just having a delete button looked awkward

Reply:
That's a good question (Related to changing passwords). I'm not sure what the bet practice is relating to this.

Reply:
done. I've added this change directly to this PR, please check it out.